src/test/ui/abi/issues/issue-22565-rust-call.stderr

```diff
@@ -1,26 +1,35 @@
-error: functions with the "rust-call" ABI must take a single non-self argument that is a tuple
-  --> $DIR/issue-22565-rust-call.rs:3:1
+error[E0277]: functions with the "rust-call" ABI must take a single non-self tuple argument
+  --> $DIR/issue-22565-rust-call.rs:3:23
    |
 LL | extern "rust-call" fn b(_i: i32) {}
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-error: functions with the "rust-call" ABI must take a single non-self argument that is a tuple
-  --> $DIR/issue-22565-rust-call.rs:9:5
+   |                       ^ `i32` is not a tuple
```

Comment:
This span is weird.

Reply:
I think we could probably pass the arg span into the obligation cause or something to make this better